configure.py

```diff
@@ -1869,6 +1883,16 @@ def query_seastar_flags(pc_file, link_static_cxx=False):
         for binary in sorted(build_artifacts):
             if binary in other:
                 continue
+            if binary in wasms:
+                src = wasm_deps[binary]
+                wasm = binary[:-4] + '.wasm'
+                if src.endswith('.rs'):
+                    f.write('build $builddir/{}/{}: rust2wasm.{} {} | test/resource/wasm/rust/Cargo.lock\n'.format(mode, wasm, mode, src))
+                    f.write('   example = {}\n'.format(binary[binary.rindex('/')+1:-4]))
+                else:
+                    f.write('build $builddir/{}/{}: c2wasm.{} {}\n'.format(mode, wasm, mode, src))
+                f.write('build $builddir/{}/{}: wasm2wat.{} $builddir/{}/{}\n'.format(mode, binary, mode, mode, wasm))
```

Comment:
New code should use f-strings, not format().

Reply:
Ok